mixins/demo/localize-mixin.html

```diff
@@ -6,14 +6,90 @@
 		<link rel="stylesheet" href="../../components/demo/styles.css" type="text/css">
 		<script type="module">
 			import '../../components/demo/demo-page.js';
-			import './localize-test.js';
 		</script>
 	</head>
 	<body unresolved>
 		<d2l-demo-page page-title="localize-mixin">
+
+			<h2>localize()</h2>
 			<d2l-demo-snippet>
-				<d2l-test-localize name="Bill"></d2l-test-localize>
+				<script type="module">
+import { html, LitElement } from 'lit';
```

Comment:
That would appear to be an issue of not having `--node-resolve` on the s3 version. It works fine locally with the web server.

Reply:
I removed that test file because it was being used for two different things and had lost it's original purpose, which was to test synchronous langterm retrieval. So I rewrote the demo page to include the relevant code for using the mixin in the `demo-snippet` (not very helpful without it), and modified the tests to include both sync and async setups internally.

Reply:
I'm going to re-introduce a couple files for these, but we won't use them from the tests. :)

Reply:
Sorry, not "not having `--node-resolve`". I mean, it not functioning the same as the local web server. I haven't looked at the way the s3 version works.

Reply:
It's unfortunate that the mixins can't have more useful demo code displayed, since they are not just element APIs.

Reply:
For sure, yeah.

Reply:
Yep I chatted briefly with @dlockhart about this - `web-dev-server` is doing some extra magic that `rollup` doesn't (and can't easily?) do when building.